lib/NetworkManager.js

```diff
@@ -281,8 +285,9 @@ class Request {
    * @param {string} url
    * @param {string} resourceType
    * @param {!Object} payload
+   * @param {Puppeteer.Frame} frame
```

Comment:
nit: question mark is missing for the frame jsdoc annotation

Reply:
Did it!
https://github.com/GoogleChrome/puppeteer/pull/1646/commits/577117b362e311fcc7e471cb82b8c11ac21c575e